src/main/java/org/spongepowered/api/world/storage/WorldProperties.java

```diff
@@ -105,6 +106,13 @@
     String getWorldName();
 
     /**
+     * Gets the directory the files for this world are located.
+     *
+     * @return The directory the files are located in
+     */
+    File getWorldDirectory();
+
+    /**
```

Comment:
@simon816 Getting the literal location of the world files is important. Whether that be a directory, or some other convoluted madness.

Who's taking on the task of creating this service?

Just an additional two-cents, however - If we're going to go this route of allowing the potential for world-storage that isn't flat-file, that means whatever service defines the world location would also need a standardized way for storing additional data provided by plugins.